common/utils/src/main/java/org/apache/spark/internal/LoggerFactory.java

```diff
@@ -19,6 +19,11 @@
 
 public class LoggerFactory {
 
+  public static Logger getLogger(String name) {
```

Comment:
`NettyLogger` use it
<img width="496" alt="image" src="https://github.com/apache/spark/assets/15246973/e11bf41b-9dd8-4fcf-81a2-77986f15d8b1">